library/src/webapp/content/copyright/custom.copyright.1.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd">
+<html>
+	<head>
+		<meta http-equiv="Content-type" content="text/html;charset=UTF-8">
+		<meta http-equiv="Refresh" content="0:URL=https://fairuse.stanford.edu/overview/faqs/copyright-ownership/#what_are_the_exceptions_to_the_rule_that_the_creator_of_a_work_owns_the_copyright">
```

Comment:
Should there be some comments in these Stanford files to make it clear they are just examples?

Reply:
I've provided some comments at the top of the files.